src/events.jl

```diff
@@ -45,10 +45,10 @@ Displaz.event_loop(
 end
 ```
 """
-function event_loop(callback::Function, event_list...)
+function event_loop(plotobj::DisplazWindow, callback::Function, event_list...)
```

Comment:
I'd just put it second (ie, flip the order of `DisplazWindow` and `Function` arguments).  Then we'll have

```julia
event_loop(window, KeyEvent("a")=>Void) do event, arg
    @show event,arg
end
```